designs/developer/operator-pipeline-45/pipelines.md

```diff
@@ -0,0 +1,244 @@
+
+- When navigating to the logs tab, the most recently started task will be selected, and the UI will auto-update, selecting the most recently started task as the pipeline run executes.
+- Clicking on a task will move to the Logs tab of the Pipeline Run with the task selected, showing the logs of all associated steps.
+- A Download link is to download the step logs.
+
+## Pipeline Run Actions
+Supported pipeline run actions are:
+1. Restart
+1. Cancel
+1. Delete Pipeline Run
+
+Pipeline Run actions can be accessed from the kebab menu in the Pipeline Runs resources tab of the Pipeline Detail
+![PLR Details Actions](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/PLR-Details-Actions.png)
+
+Pipeline actions can be accessed from the Actions menu on the Pipeline details view.
+![PLR List View Actions](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/PL-Details-PLR-Actions.png)
+
+### Restart
+No dialog is needed to restart the pipeline run.
+
+### Cancel
+No dialog is needed to cancel the pipeline run.
+
+### Delete Pipeline Run
+A delete confirmation modal is displayed when the user deletes a pipeline
+![PLR Delete Confirmation](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/PL-Details-PLR-DeleteConfirmation.png)
+
+# Task Status Component
+## Overview
+- The task status component is  a 100% stacked bar chart (Implementation note: use PF4/Victory to remain aligned with OpenShift console strategy).
+- Last status represents the tasks of the last pipeline run.
+- Each segment of the chart represents the group of tasks which are the same status.
+- Task segments are in the following order:
+    - Succeeded
+    - Failed
+    - Cancelled
+    - Skipped
+    - Running
+    - Pending
+- Hovering over the widget will show a tooltip indicating the number of tasks in each of the task statuses
+- The height of the task status component should remain constant and similar in height to associated text.
+- Clicking on any part of the task status widget will navigate to the Log Tab of the pipeline run details view.
+- Use a 3 px separator between segments
+
+# Pipeline Visualization
+## Overview
+This visualizes all tasks associated with the pipeline.
+
+![PL Visualization](http://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-42/img/PLVisualization.png)
+
+
+# Pipeline Run Visualization
+- This visualizes the status of the tasks and associated steps of the pipeline run.
+- Tasks can be asynchronous and are represented by bubbles in the visualization
+- Steps are synchronous and are represented as line segments inside the task bubbles
+- In 4.2, only the tasks which have started will be able to be shown in the Pipeline Run visualization.
+
+![PLR Visualization](img/PLRVisualization.png)
```

Comment:
if you keep the relative path here, you'll have to add the image in the designs/developer/operator-pipeline-45/img directory, or you can point back to the 4.4 image.

Reply:
OK. I will update this one to an absolute path.